docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,312 @@
+======================
+HTTP client properties
+======================
+
+``http-client.https.hostname-verification``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Verifies that the server hostname matches the server certificate.
+
+``http-client.http2.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:**
+
+.. No default value?
```

Comment:
For this property and others alike. Just confirming that there is no default value.

Reply:
It doesn't look like there is. For this and others, if there's no specification in HttpClientConfig, I believe that means there is no default and it must be handled explicitly.